mlflow/utils/databricks_utils.py

```diff
@@ -173,6 +173,14 @@ def get_job_type():
         return _get_context_tag("jobTaskType")
 
 
+def get_command_run_id():
+    try:
+        return _get_command_context().commandRunId().get()
+    except Exception:
+        # older runtimes may not have the commandRunId available
+        return "unknown_command_run_id"
```

Comment:
Thanks for the change! It might be cleaner to just not pass a command run ID in this case. We could return `None`, and then in the caller check whether the command run ID is defined before including it in request headers. 

So maybe we could rename this method to `get_command_run_id_if_exists` and update the logic here to:
```suggestion
       # Older runtimes may not have the commandRunId available
        return None
```

And then in ``mlflow/tracking/request_header/databricks_request_header_provider.py``:

```
        command_run_id = databricks_utils.get_command_run_id_if_exists()
        if command_run_id is not None:
            request_headers["command_run_id"] = command_run_id
```

Reply:
i see that `get_cluster_id` can also return None, so I'm not gonna rename it to keep it simple